subprojects/launcher/src/integTest/groovy/org/gradle/launcher/daemon/DaemonLifecycleSpec.groovy

```diff
@@ -296,7 +295,6 @@ class DaemonLifecycleSpec extends AbstractDaemonLifecycleSpec {
 
     // GradleHandle.abort() does not work reliably on windows and creates flakiness
     @Requires(UnitTestPreconditions.NotWindows)
-    @Ignore("TODO: Fix GradleHandle.abort() so that it doesn't hang")
```

Comment:
I guess you are hoping the flakiness is Windows-only and this was left behind unintentionally?